repos/roslyn.proj

```diff
@@ -27,15 +30,26 @@
     <NuSpecFiles Include="$(ProjectDirectory)/src/NuGet/Microsoft.CodeAnalysis.Compilers.nuspec" />
     <NuSpecFiles Include="$(ProjectDirectory)/src/NuGet/Microsoft.CodeAnalysis.VisualBasic.nuspec" />
     <NuSpecFiles Include="$(ProjectDirectory)/src/NuGet/Microsoft.Net.Compilers.netcore.nuspec" />
-    <NuSpecFiles Include="$(ProjectDirectory)/src/NuGet/Microsoft.CodeAnalysis.Build.Tasks.nuspec" />
   </ItemGroup>
 
+  <Target Name="UpdateCLIVersion"
+          BeforeTargets="Restore">
+
+    <UpdateJson JsonFilePath="$(ProjectDirectory)/global.json"
+                PathToAttribute="sdk.version"
+                NewAttributeValue="$(SDK_VERSION)" />
+
+    <UpdateXml  XmlFilePath="$(ProjectDirectory)/build/Targets/Tools.props"
+                PathToElement="Project/PropertyGroup/dotnetSdkVersion"
+                NewElementValue="$(SDK_VERSION)" />
```

Comment:
This needs to be removed once we merge the private branch as we jus condition away this check altogether.

Reply:
This can just be removed in this PR as it's based on the dev/sourcebuild branch.

---
In reply to: [180162495](https://github.com/dotnet/source-build/pull/404#discussion_r180162495) [](ancestors = 180162495)

Reply:
Removed.  Thanks.